src/OSPSuite.Infrastructure.Import/Services/Importer.cs

```diff
@@ -113,7 +113,7 @@ public IDataSourceFile LoadFile(IReadOnlyList<ColumnInfo> columnInfos, string fi
 
       public void CalculateFormat(IDataSourceFile dataSource, IReadOnlyList<ColumnInfo> columnInfos, IReadOnlyList<MetaDataCategory> metaDataCategories, string sheetName)
       {
-         if (sheetName.IsNullOrEmpty())
+         if (sheetName == null)
             throw new UnsupportedFormatException(dataSource.Path);
```

Comment:
@georgeDaskalakis Don't understand how it's supposed to fix #1138. 
It will still throw the **UnsupportedFormatException**, right? And I don't see a reason why the CSV file from the issue is "unsupported".